Nodejs/Product/NodejsToolsVsix/NodejsToolsVsix.csproj

```diff
@@ -71,6 +71,14 @@
       <VSIXSubPath>.</VSIXSubPath>
       <Private>True</Private>
     </ProjectReference>
+    <ProjectReference Include="..\..\MigrateToJsps\MigrateToJsps.csproj">
+      <Project>{FCAE2282-8ADC-46C8-B4C8-C25A76BD9567}</Project>
+      <Name>MigrateToJsps</Name>
+      <IncludeOutputGroupsInVSIX>BuiltProjectOutputGroup%3bBuiltProjectOutputGroupDependencies%3bGetCopyToOutputDirectoryItems%3bSatelliteDllsProjectOutputGroup%3b</IncludeOutputGroupsInVSIX>
```

Comment:
Item #&#x2060;2 [above](https://github.com/microsoft/nodejstools/pull/2456#issue-1626392894) -- %3b should be ;